standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1825,6 +1825,36 @@ public enum ConfVars {
         "hive.metastore.properties.servlet.auth", "jwt",
         "Property-maps servlet authentication method (simple or jwt)."
     ),
+    ICEBERG_CATALOG_SERVLET_PATH("hive.metastore.catalog.servlet.path",
```

Comment:
That is supposed to be table-format agnostic RestCatalog

Reply:
I just realized we already use that approach in HS2: [LDAPAuthenticationFilter](https://github.com/apache/hive/blob/master/service/src/java/org/apache/hive/service/servlet/LDAPAuthenticationFilter.java)

Reply:
Another OSS https://github.com/Netflix/metacat also has similar ability-supporting a variety of different kinds of catalog.